webcomponents/tool/src/main/frontend/sakai-editor.js

```diff
@@ -0,0 +1,42 @@
+import {SakaiElement} from "./sakai-element.js";
+import {html} from "./assets/lit-element/lit-element.js";
+
+class SakaiEditor extends SakaiElement {
+    static get properties() {
+        return {
+            textAreaId: { type: String },
+            text: { type: String }
+        };
+    }
+
+    constructor() {
+        super();
+        console.log("Sakai Editor constructor");
+        this.textAreaId = 'editor';
```

Comment:
Be consistent about your quoting. The recommendation I've read is to just stick to double quotes all the time.